src/Routes/SignIn.jsx

```diff
@@ -74,6 +77,7 @@ const SignIn = ({ isLoading }) => {
           </Button>
         </div>
       </form>
+      <Toaster />
     </div>
```

Comment:
Il y a le problème suivant indiqué dans la console : 

> The label's for attribute doesn't match any element id. This might prevent the browser from correctly autofilling the form and accessibility tools from working correctly.
> To fix this issue, make sure the label's for attribute references the correct id of a form field.

Je te propose de profiter de le corriger un ajoutant dans la balise `<Input>` concernant l'email un `id` : 
```
<Input
        id="email" // Ajout de l'attribut id
```
De même pour la balise `<Input>` concernant password : 
```
<Input
        id="password" // Ajout de l'attribut id
```
Applique également ces corrections au composant SignUp.
Une fois réalisée, tu peux ajouter à ton commit l'issue #30